src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -1809,6 +1809,21 @@ public override void Accept(OperationVisitor visitor)
         }
     }
 
+    internal partial class BoundMethodToken
+    {
+        protected override OperationKind ExpressionKind => OperationKind.None;
+
+        public override void Accept(OperationVisitor visitor)
```

Comment:
This should be unreachable, correct?  Since we don't expose lowered trees through IOperation?